spec/support/vcr_cassettes/va_profile/v3/contact_information/get_address_status_error.yml

```diff
@@ -0,0 +1,55 @@
+---
+http_interactions:
+  - request:
+      method: get
+      uri: <VETS360_URL>/contact-information-hub/contact-information/v2/addresses/status/1bfcadca-fbfa-427c-8da2-96da9111f16b
+      body:
+        encoding: US-ASCII
+        string: ''
+      headers:
+        Accept:
+          - application/json
+        Content-Type:
+          - application/json
+        User-Agent:
+          - Vets.gov Agent
+        Cufsystemname:
+          - VETSGOV
+        Accept-Encoding:
+          - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+    response:
+      status:
+        code: 404
+        message: ''
+      headers:
+        Date:
+          - Fri, 10 Jan 2020 20:54:26 GMT
+        Expires:
+          - '0'
+        Vet360txauditid:
+          - 6d90a453-36ba-4cf1-a887-f910111e1d34
+        X-Frame-Options:
+          - DENY
+        Pragma:
+          - no-cache
+        X-Content-Type-Options:
+          - nosniff
+          - nosniff
+        X-Xss-Protection:
+          - 1; mode=block
+        Cache-Control:
+          - no-cache, no-store, max-age=0, must-revalidate
+        X-Ua-Compatible:
+          - IE-edge,chrome=1
+        Content-Type:
+          - application/json;charset=UTF-8
+        Content-Length:
+          - '236'
+      body:
+        encoding: UTF-8
+        string: '{"messages":[{"code":"CORE103","key":"_CUF_NOT_FOUND","severity":"ERROR","text":"The
+        tx for id/criteria 1f450c8e-4bb2-4f5d-a5f3-0d907941625a could not be found.
```

Comment:
1bfcadca-fbfa-427c-8da2-96da9111f16b does not match 1f450c8e-4bb2-4f5d-a5f3-0d907941625a